docs/network-service-wiring.md

```diff
@@ -0,0 +1,72 @@
+This document aims at identifying the workflow a network service provider (NSP) would use to manage network services (NS) with some network service wirings (NSW).
+
+The use case is a NSP who wants to provide a gold-network NS, compose by a vpngateway at the begining, and then by a firewall and a vpngateway.
```

Comment:
This is a good start, might be useful to have a picture of the intended state (people like pictures :) ).